docs/reference/contributing/target/ResetReason.md

```diff
@@ -0,0 +1,38 @@
+### Reset Reason
+
+The Reset Reason API provides a platform-independent method of checking the cause of the last system reset. Implementing the Reset Reason API is not mandatory, but by implementing it, you can check for erroneous system resets at boot time.
+
+#### Assumptions
+
+##### Defined behavior
+
+- The function `hal_reset_reason_clear` clears the reset reason registers for the next system boot.
+- The function `hal_reset_reason_get` will get the reset reason currently in the reset reason registers. This may not necessarily be the value of the reset reason registers that were set at boot time if they have been cleared since.
+
+##### Undefined behavior
+
+- The function `hal_reset_reason_get` is not guaranteed to return the same value when called repeatedly. The user should store the value for later use instead of calling the function repeatedly.
```

Comment:
Query: Would it be accurate to replace this bullet with "There is no guarantee that the function `hal_reset_reason_get` will return the same value when you call it repeatedly. Store the value for later use instead of calling the function repeatedly." for active voice and consistent person?

Reply:
Yeah, that reads better.